src/main/java/teammates/ui/output/FeedbackSessionLogType.java

```diff
@@ -0,0 +1,16 @@
+package teammates.ui.output;
+
+/**
+ * The feedback session log type.
+ */
+public enum FeedbackSessionLogType {
```

Comment:
Maybe put this into `Const.java`

Reply:
Sure!

Reply:
Sorry, I realised actually if we put if in `Const.java` won't it be inaccessible to the frontend? Should it should be put somewhere under `teammates.ui`?

Reply:
Right. If you want to sync up with the frontend, would be worth to place it in `requests`